composer.json

```
@@ -13,7 +13,7 @@
         "phpunit/phpunit": "^9.6",
         "silverstripe/frameworktest": "^1",
         "squizlabs/php_codesniffer": "^3.7",
-        "silverstripe/cms": "^5",
+        "silverstripe/cms": "^5.2",
```

```suggestion
        "silverstripe/cms": "^5",
```
Update the framework and admin dependencies in `require`, not cms in `require-dev`

---

Opps :-) Have updated